queries.sql

```diff
@@ -82,4 +82,55 @@ FROM owners o
 JOIN animals a ON o.id = a.owner_id
 GROUP BY o.full_name
 ORDER BY COUNT(a.id) DESC
-LIMIT 1;
\ No newline at end of file
+LIMIT 1;
+
+SELECT name AS animal 
+FROM animals 
+JOIN visits ON animals.id = visits.animals_id 
+WHERE date_of_visit = '2021-01-11';
+
```

Comment:
- [ ] Great work on making this query so far but this is not professional way of doing it dear 😨 Microverse wants you to make a query from just a name William Tacher and in the real life when you would be dealing with the database then you won't be given any date or anything so specific but you would be told to make a query based on a statement given to you 👍 The right and professional query for this is given below ⬇️ Kindly try to understand and make your query according to that 😊

```sql
  SELECT animals.name, vistis.visit_date
  FROM visits
  LEFT JOIN animals ON animals.id = visits.animal_id
  LEFT JOIN vets ON vets.id = visits.vet_id
  WHERE vets.name = 'William Tatcher'
  ORDER BY visits.visit_date DESC
  LIMIT 1;

```